### PR TITLE
Improve readme

### LINKED ISSUE #10: Provide an example with multiple path
Hello,

The doc misses an example with multiple playbook linting;

Right now I've tried: 

```yaml
name: CI  # feel free to pick your own name

on: pull_request

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
    # Important: This sets up your GITHUB_WORKSPACE environment variable
    - uses: actions/checkout@v2

    - name: Install dependencies
      run: ansible-galaxy install -r roles/requirements.yml --force

    - name: Lint Ansible Playbook
      # replace "master" with any valid ref
      uses: ansible/ansible-lint-action@master
      with:
        # [required]
        # Paths to ansible files (i.e., playbooks, tasks, handlers etc..)
        # or valid Ansible directories according to the Ansible role
        # directory structure.
        targets:
          - "playbook-example.yml"
          - "playbook-provisionning.yml"
```

But Github says that 

```
### ERRORED 13:29:06Z

- Your workflow file was invalid: The pipeline is not valid. .github/workflows/ci.yml (Line: 26, Col: 11): A sequence was not expected
```

### PR DESCRIPTION
**Pull request by tristanbes**
I've made some changes to the `readme.md`:

- [x]  Renamed `readme.md` to `README.md` which is more standard;
- [x] Uses `actions/checkout@v2` which is more efficient than `v1`
- [x] Added an example on how to use this action with multiple ansible files
- [x] Changed default event from `push` to `pull_request`, which is better by "default".

Fixes #10

### CODE REVIEW

**Comment by tristanbes** [comment]
ping @webknjaz

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ For example:
 ```yaml
 name: Ansible Lint  # feel free to pick your own name
 
-on: push
+on: [pull_request]
```
Comment (webknjaz) [inline README.md]: Why not have both?
```suggestion
on:
  pull_request:
  push:
```

**Comment by rajbos** [comment]
:robot: Forked the action repository over to rajbos-actions :high_voltage:
Go to [linkUrl](linkUrl) to see the action repository.

**Comment by rajbos** [comment]
:robot: Forked the action repository over to rajbos-actions :high_voltage:
Go to [linkUrl](linkUrl) to see the action repository.

**Comment by rajbos** [comment]
:robot: Forked the action repository over to rajbos-actions :high_voltage:
Go to [linkUrl](linkUrl) to see the action repository.

**Comment by rajbos** [comment]
:robot: Forked the action repository over to rajbos-actions :high_voltage:
Go to [linkUrl](linkUrl) to see the action repository.

**Pull request merged**